Joyce/index.js

```diff
@@ -0,0 +1,25 @@
+// 1
+let introSentence = "hello, my name is Anna and I am ";
+let age = 45;
+console.log(introSentence + age);
+
+//  The + operator is concatenating the string and the number
+
+// 2
+let value1 = "1005";
+console.log(parseInt(value1));
+let value2 = "10.05";
+console.log(parseFloat(value2));
+
+// 3
```

Comment:
The last solution(3rd question) is not correct. One of best solutions coud be:
let num3 = 30;
let num4 = 939;
let num5 = 40.9;

let evenNum1 = num3 % 2 ? "is odd" : "is even";
console.log(num3, evenNum1);

let evenNum2 = num4 % 2 ? "is odd" : "is even";
console.log(num4, evenNum2);

let evenNum3 = num5 % 2 ? "is odd" : "is even";
console.log(num5, evenNum3);

Reply:
here "is odd" : "is even"; I would have used "is even" : "is odd";  because we learned yesterday ==> condition ? "if condition true": "if condition false"